recipes/gstlal/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# conda-forge/conda-forge.github.io#621
+find ${PREFIX} -name "*.la" -delete
+
```

Comment:
This is missing `./00init.sh` before running `./configure`.

Reply:
@myNameIsPatrick, I don't think the `00init.sh` is required when you build from the tarballs.

Reply:
@duncanmmacleod, my bad, I didn't realize this. Ignore my previous comment.